parser/lexer.go

```diff
@@ -412,7 +412,7 @@ func startWithAt(s *Scanner) (tok int, pos Pos, lit string) {
 		s.r.incAsLongAs(isIdentChar)
 		tok, lit = doubleAtIdentifier, s.r.data(&pos)
 	} else {
-		tok = at
+		tok = int('@')
```

Comment:
Why change this?

Reply:
'@' is a single character token, `at` should be reserved for the keyword `AT`.
Even though we haven't use the keyword `AT` yet.